data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_eth_v1_beacon_blinded_blocks.json

```diff
@@ -1,92 +1,115 @@
-              "$ref" : "#/components/schemas/SignedBlindedBlockElectra"
-            }  ]
+        "application/json": {
+          "schema": {
+            "title": "SignedBlindedBlock",
+            "type": "object",
+            "oneOf": [
+              {
+                "$ref": "#/components/schemas/SignedBeaconBlockPhase0"
+              },
+              {
+                "$ref": "#/components/schemas/SignedBeaconBlockAltair"
+              },
+              {
+                "$ref": "#/components/schemas/SignedBlindedBlockBellatrix"
+              },
+              {
+                "$ref": "#/components/schemas/SignedBlindedBlockCapella"
+              },
+              {
+                "$ref": "#/components/schemas/SignedBlindedBlockDeneb"
+              },
+              {
+                "$ref": "#/components/schemas/SignedBlindedBlockElectra"
+              }
+            ]
           }
         }
       }
     },
-    "responses" : {
-      "415" : {
-        "description" : "Unsupported media-type supplied",
-        "content" : {
-          "application/json" : {
-            "schema" : {
-              "$ref" : "#/components/schemas/HttpErrorResponse"
+    "responses": {
+      "415": {
+        "description": "Unsupported media-type supplied",
+        "content": {
+          "application/json": {
+            "schema": {
+              "$ref": "#/components/schemas/HttpErrorResponse"
             }
           }
         }
       },
-      "200" : {
-        "description" : "Block has been successfully broadcast, validated and imported.",
-        "content" : { }
+      "200": {
+        "description": "Block has been successfully broadcast, validated and imported.",
+        "content": {}
+      },
+      "202": {
+        "description": "Block has been successfully broadcast, but failed validation and has not been imported.",
+        "content": {}
       },
-      "202" : {
-        "description" : "Block has been successfully broadcast, but failed validation and has not been imported.",
-        "content" : { }
+      "204": {
+        "description": "Data is unavailable because the chain has not yet reached genesis",
+        "content": {}
       },
-      "400" : {
-        "description" : "Unable to parse request body.",
-        "content" : {
-          "application/json" : {
-            "schema" : {
-              "$ref" : "#/components/schemas/HttpErrorResponse"
+      "400": {
+        "description": "Unable to parse request body.",
+        "content": {
+          "application/json": {
+            "schema": {
+              "$ref": "#/components/schemas/HttpErrorResponse"
             }
           }
         }
       },
-      "503" : {
-        "description" : "Beacon node is currently syncing.",
-        "content" : {
-          "application/json" : {
-            "schema" : {
-              "$ref" : "#/components/schemas/HttpErrorResponse"
+      "503": {
+        "description": "Service unavailable",
```

Comment:
Drawing your attention to 503 response. For APIs with 503 being implemented the description has changed.